src/components/ClauseComponent.js

```diff
@@ -81,8 +86,35 @@ function ClauseComponent(props) {
         .reduce((props, node) => ({ ...props, ...node }), {})
     });
 
+    let listData = {};
+    const findListVariables = (node) =>{
+      if(node.type === 'ul_list'){
```

Comment:
What about 'ol_list' ?

Reply:
ooh, I was thinking this feature was meant for only for "ul" since the issue was called "Bulleted List Variables", should I extend it to take care are of "ol" too?